mgmt/Cargo.toml

```diff
@@ -17,6 +17,7 @@ bolero = ["dep:bolero", "interface-manager/bolero", "id/bolero", "net/arbitrary"
 [dependencies]
 # internal
 gateway_config = { workspace = true }
+hyper = { version = "0.14", features = ["server", "http1", "tcp", "stream"] }
```

Comment:
Please set the version in the toplevel Cargo.toml, as for the other dependencies